packages/components/typedoc-internal.js

```diff
@@ -4,17 +4,19 @@
  * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
  */
 module.exports = {
-    out: './docs',
+    name: "@labkey/components (internal)",
+    readme: "none",
     theme: "default",
-    mode: "modules",
+    mode: "file",
+    includeVersion: true,
+    ignoreCompilerErrors: true,
     exclude: [
         "**/*+(wrapper|wrapper-dom|.spec|.stories).ts*",
         "**/stories/**",
         "**/__snapshots__/**",
-        "**/__mocks__/**"
+        "**/__mocks__/**",
+        "**/test/**",
     ],
     externalPattern: "**/node_modules/** ",
     excludeExternals: true,
-    excludePrivate: true,
```

Comment:
We're no longer excluding private.  Is that intentional?  Will there be useful docs produced for private things?

Reply:
I don't think I intended on removing that. I'll add it back. Good catch.